code/game/machinery/transformer.dm

```diff
@@ -2,24 +2,31 @@
 	name = "\improper Automatic Robotic Factory 5000"
 	desc = "A large metallic machine with an entrance and an exit. A sign on \
 		the side reads, 'human go in, robot come out'. The human must be \
-		lying down and alive. Has to cooldown between each use."
+		lying down and alive. Has a cooldown between each use."
 	icon = 'icons/obj/recycling.dmi'
 	icon_state = "separator-AO1"
 	layer = ABOVE_ALL_MOB_LAYER // Overhead
 	plane = ABOVE_GAME_PLANE
 	density = FALSE
 	active_power_usage = BASE_MACHINE_ACTIVE_CONSUMPTION * 5
-	var/transform_dead = 0
-	var/transform_standing = 0
-	var/cooldown_duration = 600 // 1 minute
-	var/cooldown = 0
+	/// Whether this machine transforms dead mobs into cyborgs
+	var/transform_dead = FALSE
+	/// Whether this machine transforms standing mobs into cyborgs
+	var/transform_standing = FALSE
+	/// How long we have to wait between processing mobs
+	var/cooldown_duration = 60 SECONDS
+	/// Whether we're on cooldown
+	var/cooldown = FALSE
+	/// How long until the next mob can be processed
 	var/cooldown_timer
+	/// The created cyborg's cell chage
 	var/robot_cell_charge = 5000
+	/// The visual countdown effect
 	var/obj/effect/countdown/transformer/countdown
+	/// Who the master AI is that created this factory
 	var/mob/living/silicon/ai/masterAI
```

Comment:
want to change this var name while you're here? 🥺

Reply:
What do you suggest?

Reply:
master_ai